config/image.go

```diff
@@ -35,6 +35,9 @@ type ImageConfig struct {
 	// Dockerfile The path to the ``Dockerfile`` used to build the image. This
 	// path is relative to the **context**.
 	Dockerfile string
+	// Dobifile is a in house version of a dockerfile
+	// type: mapping ``[Valid Docker command]: value``
+	Dobifile []map[string]string
```

Comment:
I agree we need some other name for this, but I think `dobifile` is going to be confused with the `dobi.yaml`.

I'm thinking maybe `content:` is a better name, since it's the "content" of the Dockerfile.